src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -956,8 +956,8 @@ def service_action(self, action, service_name):
         providing the logical service).
 
         :param action: one of "start", "stop", "restart", "redeploy", "reconfig"
-        :param service_type: e.g. "mds", "rgw", ...
-        :param service_name: name of logical service ("cephfs", "us-east", ...)
+        :param service_name: service_type + '.' + service_id
+        (e.g. "mon", "mgr", "mds.mycephfs", "rgw.realm.zone", ...)
         :rtype: Completion
```

Comment:
/home/jenkins-build/build/workspace/ceph-pr-docs/src/pybind/mgr/orchestrator/_interface.py:docstring of orchestrator.Orchestrator.service_action:8: WARNING: Field list ends without a blank line; unexpected unindent.

https://jenkins.ceph.com/job/ceph-pr-docs/29817/consoleFull#-85362248744e9240e-b50a-4693-bac0-8a991bac86ac